BlazorCore/DSD.MSS.Blazor.Components.Core/Models/CommentDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DSD.MSS.Blazor.Components.Core.Models
+{
+    class CommentDTO
+    {
+        public string CommentText { get; set; }
+
+        public DateTime? CommentTime { get; set; }
```

Comment:
Is this time submitted? time created? time updated? I've posted a suggestion based on my assumption.
```suggestion
        public DateTime? CommentCreatedDate { get; set; }
```